_applications/var.md

```diff
@@ -29,97 +29,83 @@ sophisticated manner.
 
-Once the `var-notebook` service is visible within the developer console, select
-"Create Route" and create a public route to the `var-notebook` service,
-targeting port 8888. The public route you create will depend on your DNS
-configuration, but you can use the [xip.io
-service](https://access.redhat.com/solutions/2141701) for development and
-testing.
+<iframe src="https://player.vimeo.com/video/194528216" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+<p><a href="https://vimeo.com/194528216">Value-at-risk notebook demo in OpenShift</a>.</p>
 
-<h1 id="usage">Launching and Using the Notebook</h1>
+# Appendix: running against a cluster
 
-Verify that the `var-notebook` service is running and that your Spark cluster
-is running by looking at the developer console. Once each is up and functional,
-you can use the notebook by visiting the route you defined earlier in your
-browser.
+While it certainly isn't necessary for the amount of data we're processing in this project, you could also run your simulations against a Spark cluster.  We'll show you how to do that now.
 
-Interacting with the Jupyter notebook is very simple. Select a cell with Python
-code and execute it by pressing the "run cell" button in the toolbar. Before
-you get started, you'll need to make sure you're able to connect to your Spark
-cluster: in the very first code cell, find the line that looks like this:
+## Clustered installation prerequisites
 
-`spark = SparkSession.builder.master("spark://sparky:7077").getOrCreate()`
+The easiest way to get a Spark cluster going in your OpenShift project is to use [the
+OpenShift template](https://radanalytics.io/get-started#quickstart-with-oshinko)
+provided on the "Getting Started page". This provides a streamlined installation for
+the Oshinko service, which manages Spark clusters in OpenShift projects.
 
-and replace `sparky` with whatever name you chose for your Spark cluster.
+We'll need to use a special Spark worker image for this application. This worker image
+has a small amount of historical stock return data stored in its filesystem. For a real
+application, you'd want to get data from persistent storage, from a database, or from
+an object store service. For a self-contained tutorial demo, though, it's much easier
+to package up a small amount of data where each worker can get it.
 
-You can edit the code in the notebook cells and re-run them, and results from
-cells you've already run will be available to new cells. The notebook interface
-provides a great way to experiment with new techniques. Try it out!
+In order to get this special Spark worker image, we'll need to pass a special option 
+to the Oshinko template when we install it in our project.  Set the `OSHINKO_CLUSTER_IMAGE` variable to `radanalyticsio/workshop-notebook:worker`.  This image is based on the default `radanalyticsio/openshift-spark` image, but it also includes our historical stock data.
```

Comment:
while i do think the clustered execution is cool, in this case it just seems fraught with troubles. we are advising a design pattern that we acknowledge isn't that great, and i'm not sure adding the cluster to this notebook adds much in the performance area.

it might be a nice opportunity to talk about why adding a cluster to this notebook would troublesome though. like, a teachable moment about using data files in your spark applications?